app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileFragment.java

```diff
@@ -1,31 +1,88 @@
 package com.cmput301f20t21.bookfriends.ui.profile;
 
+
+import androidx.annotation.NonNull;
 import androidx.lifecycle.ViewModelProvider;
 
+import android.content.Intent;
 import android.os.Bundle;
 
-import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import androidx.fragment.app.Fragment;
 
 import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
 
 import com.cmput301f20t21.bookfriends.R;
+import com.cmput301f20t21.bookfriends.ui.login.LoginActivity;
 
-public class ProfileFragment extends Fragment {
+
+public class ProfileFragment extends Fragment implements ProfileEditDialog.ProfileEditListener {
 
     private ProfileViewModel mViewModel;
+    private TextView emailAddress;
+    private TextView phoneNumber;
+    private ImageView editProfile;
 
     public static ProfileFragment newInstance() {
         return new ProfileFragment();
     }
 
+    @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                              @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_profile, container, false);
+        View view = inflater.inflate(R.layout.fragment_profile, container, false);
+        // TODO for other profile just use .setVisibility(View.GONE); to hide login button and edit icon
+        emailAddress = view.findViewById(R.id.email);
+        phoneNumber = view.findViewById(R.id.phone);
+        editProfile = view.findViewById(R.id.image_edit);
+        //click on the logout button, bring back to the login activity
+        Button logout = view.findViewById(R.id.logout_button);
+        logout.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(getActivity(), LoginActivity.class);
+                startActivity(intent);
```

Comment:
If I clicked logout and then clicked the back button on the phone emulator it will bring me back to the profile page

might want to call getActivity().finish() after you start the login activity